core/src/lib/services/internal/CoreStitchServiceImpl.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  CoreStitchService,
+  StitchAuthRequestClient,
+  StitchServiceRoutes
+} from "../..";
+import { StitchAuthDocRequest } from "../../internal/net/StitchAuthDocRequest";
+import Method from "../../internal/net/Method";
+
+export default class CoreStitchServiceImpl implements CoreStitchService {
```

Comment:
[nit] Rename `CoreStitchService` and `StitchService` to `CoreStitchServiceClient` and `StitchServiceClient`